config/babel/.babelrc

```diff
@@ -1,4 +1,3 @@
 {
-  "presets": ["react", "react-native"],
-  "plugins": ["lodash"]
```

Comment:
My understanding is that it would be very unlikely the babel plugin would affect behavior. And this runs in dev, so if the issue doesn't crop up in dev, it's not this plugin.